src/utc_offset.rs

```diff
@@ -8,6 +8,9 @@ use crate::{
 };
 use core::fmt::{self, Display};
 
+#[cfg(target_arch = "wasm32")]
+use stdweb::js;
```

Comment:
_Major_ point here: `stdweb` does not work on any wasm target. wasmtime, for example, would have no idea what to do.

Reply:
I agree it's a problem. The trouble is, wasm is a very generic standard that defines no default interface to interact with the outside world. I will check if it is possible to narrow this down to a javascript environment.